tools/token_generator/main.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+var jwtSecret = []byte(os.Getenv("JWT_SECRET"))
+
+type TokenClaims struct {
+	Version string `json:"version"`
+	jwt.RegisteredClaims
+}
+
+func GenerateJWT() (string, error) {
+	claims := TokenClaims{
+		Version: "0.3.0",
+		RegisteredClaims: jwt.RegisteredClaims{
+			ExpiresAt: jwt.NewNumericDate(time.Now().Add(time.Hour * 24)),
+			IssuedAt:  jwt.NewNumericDate(time.Now()),
+			Issuer:    "testuser",
```

Comment:
The issuer should be: heureka.

Reply:
I have changed it to heureka, and added Subject as a name of the user. Is it ok? if not, please tell me what fields should be used to keep what value in your opinion.

Reply:
This is ok for now. Thanks!